.github/workflows/run-end-to-end.yml

```diff
@@ -115,8 +115,8 @@ jobs:
         DD_API_KEY: ${{ secrets.DD_API_KEY }}
         AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
         AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-        AWS_DEFAULT_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
-        AWS_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
+        AWS_REGION: us-east-1
```

Comment:
Can you set a defaulting in the code to `us-east-1`, and taking the value form the env var if and only if it's set ? 

With that, the test would work without the need of definining it, unless you really need to overwrite it.

Reply:
yes, done

Reply:
You mean, already done, or in a commit not yet pushed ?

Reply:
My point was for the scenario that require this value, get it in python by : 

```python
   value = os.environ.get("AWS_REGION", "us-east-1")
```

With that, it reduces the amount of env var to add in all `yml` files.

Reply:
i don't get the resolution here. IIUC with the proposed changes: 

* `AWS_REGION` and `AWS_DEFAULT_REGION` env var are set in the CI
* they are used only by buddy container
* they are not used at all by weblog container, as `"us-east-1"` is used harcoded.

My change request was : 

* do not set at all `AWS_REGION` and `AWS_DEFAULT_REGION` in CI yml files
* in the code, use `os.environ.get("<name>", "us-east-1")` -> with that, `us-east-1` will be used by default. But if, for some reason, another value is needed, we will be able to set it

Reply:
In my opinion, I think its easiest to still set `AWS_REGION` and `AWS_DEFAULT_REGION ` as environment variables, and not hardcode it where needed. The reason for this is because AWS clients always need a region. In this case, they default to `AWS_REGION` or `AWS_DEFAULT_REGION`. If we are using many different AWS clients throughout the code, it's easiest to keep this 1 environment variable, as opposed to engineers needing to set the AWS region manually within their code. It's also quite common to always set the `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_REGION` all together. Please see [here](https://docs.aws.amazon.com/cli/v1/userguide/cli-configure-envvars.html#:~:text=%24%20export%20AWS_ACCESS_KEY_ID%3DAKIAIOSFODNN7EXAMPLE%0A%24%20export%20AWS_SECRET_ACCESS_KEY%3DwJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY%0A%24%20export%20AWS_DEFAULT_REGION%3Dus%2Dwest%2D2). And please let me know if I am misunderstanding.

Reply:
Additionally, just to be clear, the env variables are needed for ... :
- Weblogs for testing with AWS
- Buddies for testing propagation with weblogs for AWS
- Test Runner for cleaning up the used AWS resources after the test run.